### PR TITLE
Only show most recent iteration at the start of discussion posts

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/v3-project-management/issues/119.

### CODE REVIEW

**Review by iHiD — APPROVED** [review]
Nice simple solution :)

**Comment by iHiD** [comment]
Tests are failing here too. There's something really brittle in the mentor tests.

File: test/javascript/components/mentoring/discussion/DiscussionPostList.test.js [hunk 1]
```diff
@@ -96,6 +103,100 @@ test('displays all posts', async () => {
...
+      name: 'Discussion post list loading indicator',
+    })
+  )
+
+  expect(screen.queryByText('Iteration 1')).not.toBeInTheDocument()
+  expect(screen.getByText('Iteration 2')).toBeInTheDocument()
+
+  server.close()
+})
+test('shows latest ieration if there are no posts', async () => {
```
Comment (ErikSchierboom) [inline test/javascript/components/mentoring/discussion/DiscussionPostList.test.js]:
```suggestion
test('shows latest iteration if there are no posts', async () => {
```

File: test/javascript/components/mentoring/discussion/DiscussionPostList.test.js [hunk 2]
```diff
@@ -96,6 +103,100 @@ test('displays all posts', async () => {
   server.close()
 })
 
+test('shows first ieration with posts', async () => {
```
Comment (ErikSchierboom) [inline test/javascript/components/mentoring/discussion/DiscussionPostList.test.js]:
```suggestion
test('shows first iteration with posts', async () => {
```

**Review by ErikSchierboom — APPROVED** [review]
Nice!

**Pull request merged**